invokeai/frontend/web/src/app/store/middleware/listenerMiddleware/listeners/modelsLoaded.ts

```diff
@@ -204,3 +205,23 @@ const handleSpandrelImageToImageModels: ModelHandler = (models, state, dispatch,
 
   dispatch(upscaleModelChanged(null));
 };
+
+const handleSimpleUpscaleModels: ModelHandler = (models, state, dispatch, _log) => {
```

Comment:
I would just put this log in the `handleSpandrelImageToImageModels` listener

Reply:
ModelHandler requires a log, or do you mean just adding all the handleSimpleUpscaleModels logic to handleSpandrelImageToImageModels?